deployments/epinio.go

```diff
@@ -133,6 +133,17 @@ func (k Epinio) apply(ctx context.Context, c *kubernetes.Cluster, ui *termui.UI,
 	}
 
 	issuer := options.GetStringNG("tls-issuer")
+	if !internalIssuer(issuer) {
+		found, err := c.ClusterIssuerExists(ctx, issuer)
+		if err != nil {
+			return err
+		}
+		if !found {
+			ui.Exclamation().Msgf("Specified cluster issuer '%s' is missing. Please create it first.", issuer)
```

Comment:
This is printed too lat and it's not even failing the installation. I used a non-existent issuer and the installation went on as if the issuer was there. I think we should rather verify the existence of the issuer before we even start deploying the deployments. This way the user can fix the problem and then run the install command again without having to uninstall Epinio first.

Reply:
I moved the check into a 'PreInstallCheck'